src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -38,13 +38,13 @@ public void testIsFull() {
 		}
 
 		boolean result = rhymer.isFull();
-		Assert.assertEquals(true, result);

```

Comment:
Is it a correct approach to fixing errors showed by tests? If STACK_CAPACITY is 12 then 12 elements should be added to the stack and it should be full. Incorrect (test) fix.